src/rgw/rgw_sal_sfs.h

```diff
@@ -348,8 +348,11 @@ class SFStore : public Store {
     db_binfo.binfo.creation_time = ceph::real_clock::now();
     db_binfo.binfo.placement_rule = placement_rule;
     db_binfo.binfo.zonegroup = zonegroup_id;
+    if (pquota_info) {
+      db_binfo.binfo.quota = *pquota_info;
+    }
 
-    RGWObjVersionTracker& objv_tracker = info.objv_tracker;
+    RGWObjVersionTracker& objv_tracker = db_binfo.binfo.objv_tracker;
```

Comment:
Regarding this... I've created a bug (https://github.com/aquarist-labs/s3gw/issues/103) because we'd need to implement the logic of version tracking in metadata.

It works here because we're really doing nothing aside from storing the initial value.

I also suspect we should assign the value coming from `info` to consume version tracking instead of initialising every time, but that will possibly arise when we implement the version tracking logic. (and also I'm not 100% sure if we need it for buckets here or initialising when creating the bucket is fine)

Have you checked what's coming in the `info` value? I guess it's already an initialised value. 

So if assigning what's coming from `info` is breaking this PR I'm happy to leave it as it is and change it when logic is implemented (if needed).. if it's not breaking it I would store whatever is received in the `info` variable.

Reply:
The value coming from info is `empty`. 
The value must be computed initially in `create` function.
I think a value here was needed for the UI to work.

Reply:
what if it's a versioned bucket? Is the value still empty?

Reply:
The computation of a version id is always done in `create` function.
This has not been introduced with this PR btw, if we want to do changes to this behavior I suggest do it on another dedicated one.

Reply:
That's not what I meant. The bucket versioning is enabled on a bucket by the user. I would assume that this `objv_tracker` only makes sense when the bucket versioning is enabled. While you are getting an empty info in this case, might not mean it's empty if bucket versioning is enabled. Hence my question.

Reply:
can we move these checks for another issue?

Reply:
I don't think we can. We need to make sure this is not causing a problem.

Reply:
Keep in mind that if this is causing problems it already does since this [PR](https://github.com/aquarist-labs/ceph/pull/27) has been merged.
but ok, no problem for me, let's delve into the problem.

Reply:
I don't know if what has been merged is causing a problem, but the worry is that this specific change, on this specific PR, may be introducing a problem. Keep in mind that this particular change is essentially squashing whatever `objv_tracker` is coming from the `info` function parameter with an empty one (the one you are creating) somewhere down the line.

I don't think it's unreasonable to expect validation on this change before merging.

Reply:
understand and I agree; better a check more than one check less. ;)

Reply:
removed bucket version related code and columns.